pandas/tests/series/test_operators.py

```diff
@@ -94,24 +94,25 @@ def test_logical_operators_int_dtype_with_int_scalar(self):
     def test_logical_operators_int_dtype_with_float(self):
         # GH#9016: support bitwise op for integer types
         s_0123 = Series(range(4), dtype="int64")
-
-        with pytest.raises(TypeError):
+        msg = r"cannot compare a dtyped \[int64\] array with a scalar of type \[float\]"
```

Comment:
why are you changing things here?

Reply:
apologies, this shouldn't have been a change here as I was working on another branch working on the bare pytest_raises issue.